README.md

```diff
@@ -1,12 +1,21 @@
+The LibreSignal project never set up its own [Signal-Server](https://github.com/WhisperSystems/Signal-Server), but instead used the Open Whisper Systems servers. In May 2016, when asked if he was OK with LibreSignal using the Open Whisper Systems servers, [Moxie Marlinspike replied](https://github.com/LibreSignal/LibreSignal/issues/37#issuecomment-217211165) that he was not OK with this due to the operational costs of supporting a third-party client. A few days later, Open Whisper Systems published [their stance on federation](https://whispersystems.org/blog/the-ecosystem-is-moving/), which meant that LibreSignal users would have had to get their contacts to switch to LibreSignal if the LibreSignal project had decided to set up its own Signal-Server. However, Marlinspike [also said](https://github.com/LibreSignal/LibreSignal/issues/37#issuecomment-226646872) that he would be willing to consider "a clean, well written, and well tested" pull request that would add WebSocket support to Signal so that it could function on Google-free devices. A pull request was [eventually submitted](https://github.com/WhisperSystems/Signal-Android/pull/5962), and WebSocket support was added to Signal on [February 20, 2017](https://github.com/WhisperSystems/Signal-Android/commit/1669731329bcc32c84e33035a67a2fc22444c24b).
 
-To be clear, Signal relying on GCM is only an issue for people who use a custom Android ROM without Google Play Services. For the vast majority of people who do have Google Play on their phone, this issue is completely irrelevant.  Signal is designed so that [GCM is only used for a wakeup event and never sees any of the encrypted Signal messages](http://support.whispersystems.org/hc/en-us/articles/215843598-Why-GCM-Does-Google-see-my-messages-).
+Signal officially supports Google-free devices as of February 20, 2017. However, before Open Whisper Systems will officially distribute Signal outside of the Google Play Store, [they would like](https://github.com/WhisperSystems/Signal-Android/commit/1669731329bcc32c84e33035a67a2fc22444c24b#commitcomment-20980219):
 
-There are two ways in which you can help make the pull request. If you are able to contribute code, check out [this page about contributing code to Signal-Android](https://github.com/WhisperSystems/Signal-Android/blob/master/CONTRIBUTING.md). If you are not able to contribute code, consider contributing to the aforementioned bounty.
+>1. A small well-written library that we can link into Signal which does crash reporting, checking for updates (and posting notifications for the user), and reporting basic stats.
```

Comment:
>We do have ACRA for crash reporting. https://github.com/ACRA/acra

There's a discussion about this [here](https://whispersystems.discoursehosting.net/t/how-to-get-signal-apks-outside-of-the-google-play-store/808). Do you think we should link to that discussion in the README as well?